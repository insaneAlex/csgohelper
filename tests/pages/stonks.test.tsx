import {render} from '@testing-library/react';
import Stonks from '../../pages/stonks';

jest.mock('../../src/services', () => ({}));
jest.mock('next/router', () => ({useRouter: () => ({push: jest.fn()})}));
jest.mock('react-redux', () => ({useDispatch: () => jest.fn(), useSelector: jest.fn()}));

describe('Stonks', () => {
  it('should render correctly', () => {
    const {container} = render(<Stonks />);
    expect(container).toMatchSnapshot();
  });
});
