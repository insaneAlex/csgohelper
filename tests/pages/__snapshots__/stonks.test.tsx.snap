// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stonks should render correctly 1`] = `
<div>
  <div
    class="alert"
  >
    Coming soon
  </div>
</div>
`;
